TestRunner/TestRunner.swift

```diff
@@ -134,9 +135,9 @@ open class TestRunner: NSObject {
         return !failed
     }
     
-    func createOperation(_ deviceFamily: String, simulatorName: String, deviceID: String, tests: [String], retryCount: Int = 0, launchRetryCount: Int = 0) -> TestRunnerOperation {
-        let operation = TestRunnerOperation(deviceFamily: deviceFamily, simulatorName: simulatorName, deviceID: deviceID, tests: tests, retryCount: retryCount, launchRetryCount: launchRetryCount)
-        operation.completion = { status, simulatorName, failedTests, deviceID, retryCount, launchRetryCount in
+    func createOperation(_ deviceFamily: String, simulatorName: String, deviceID: String, tests: [String], retryCount: Int = 0, launchRetryCount: Int = 0, needsExternalDisplay: Bool = false) -> TestRunnerOperation {
```

Comment:
I think you can just completely remove any changes related to `needsExternalDisplay` in this class.